Emby.AutoOrganize/Emby.AutoOrganize.csproj

```diff
@@ -37,8 +37,7 @@
   <ItemGroup>
     <PackageReference Include="Jellyfin.Controller" Version="10.*" />
     <PackageReference Include="Jellyfin.Naming" Version="10.*" />
-    <PackageReference Include="SQLitePCL.pretty.netstandard" Version="1.0.0" />
-    <PackageReference Include="System.Memory" Version="4.5.1" />
+    <PackageReference Include="SQLitePCL.pretty.netstandard" Version="2.0.1" />
```

Comment:
- Update SQLite dependency to latest to match main project
- Remove explicit reference to System.Memory since it has been updated to latest in main project